pyproject.toml

```diff
@@ -1,3 +1,46 @@
 [build-system]
 requires = ["setuptools>42", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "quot"
+version = "4.0.0"
```

Comment:
From now on, I'd like to use semantic versions for `quot`. I'm starting these at 4.0.0, since the switch to `src` layout might be interface-breaking for some.